src/Symfony/Component/Cache/Traits/AbstractAdapterTrait.php

```diff
@@ -379,18 +378,14 @@ private function getId($key)
             }
         }
 
-        if (\is_string($key) && isset($this->ids[$key])) {
```

Comment:
Maybe the `this->ids` property can be removed entirely, as this is the only place it's read as far as I can see.